backend/drive/drive.go

```diff
@@ -3969,16 +3978,16 @@ func (f *Fs) Command(ctx context.Context, name string, arg []string, opt map[str
 			dir = arg[0]
 		}
 		return f.unTrashDir(ctx, dir, true)
-	case "copyid":
-		if len(arg)%2 != 0 {
-			return nil, errors.New("need an even number of arguments")
+	case "copyOrMoveID":
```

Comment:
I think we need to keep the "copyid" command for backwards compatibility, so I'd suggest you make this

```go
case "copyid", "moveid":
```

The command definitions will need duplicating though.